osf_tests/factories.py

```diff
@@ -271,6 +278,14 @@ class Meta:
     anonymous = False
     creator = factory.SubFactory(UserFactory)
 
+    @classmethod
+    def _create(cls, target_class, *args, **kwargs):
+        instance = super(PrivateLinkFactory, cls)._create(target_class, *args, **kwargs)
+        if instance.is_deleted and not instance.deleted:
+            instance.deleted = timezone.now()
+            instance.save()
+        return instance
+
```

Comment:
I see this is pulled from Erin's PR, but the purpose is so we don't have to go through all of our old tests and make sure projects where we're setting `is_deleted` on the onset, also have `deleted` set so we can use that instead in our tests? Phew it's going to take a bit to refactor our tests once we start using the deleted field instead!

Reply:
I realized this when I started looking through the tests. Perhaps making a ticket for just test refactoring wouldn't be a bad idea in between this PR and the actual is_deleted removal.

Reply:
yep, good idea @corbinSanders! Do you mind making that ticket, as well as a follow-up ticket for the remaining pieces of this work?